video/out/gpu/context.c

```diff
@@ -82,6 +83,9 @@ static const struct ra_ctx_fns *contexts[] = {
 #if HAVE_GL_WAYLAND
     &ra_ctx_wayland_egl,
 #endif
+#if HAVE_WAYLAND
+    &ra_ctx_wayland,
```

Comment:
If it were me I'd keep the vo/wayland and vo/wayland-dmabuf as separate patches (in this PR).